vars/os_Archlinux.yml

```diff
@@ -5,6 +5,8 @@ samba_packages:
   - samba
   - smbclient
 
+samba_vfs_packages: []
```

Comment:
I'm told that (like RedHat), Arch includes the fruit module in the 'samba' package so this should DTRT.